--- origsrc/xine-ui-0.99.6/src/fb/Makefile.am	2009-12-18 17:14:36.000000000 -0600
+++ src/xine-ui-0.99.6/src/fb/Makefile.am	2010-08-23 01:52:10.626366200 -0500
@@ -3,7 +3,7 @@ AM_CFLAGS = @XINE_CFLAGS@ $(X_CFLAGS) -D
 bin_PROGRAMS = fbxine 
 
 fbxine_DEPENDENCIES = $(top_builddir)/src/common/libcommon.a
-fbxine_LDADD = @XINE_LIBS@ $(LIRC_LIBS) -lm $(THREAD_LIBS) $(top_builddir)/src/common/libcommon.a
+fbxine_LDADD = $(top_builddir)/src/common/libcommon.a @XINE_LIBS@ $(LIRC_LIBS) -lm $(THREAD_LIBS)
 
 fbxine_SOURCES = main.c actions.c keys.c options.c callback.c stdctl.c post.c osd.c
 noinst_HEADERS = main.h actions.h keys.h options.h callback.h stdctl.h post.h osd.h
--- origsrc/xine-ui-0.99.6/src/xitk/Makefile.am	2009-12-18 17:14:33.000000000 -0600
+++ src/xine-ui-0.99.6/src/xitk/Makefile.am	2010-08-23 01:33:16.780754400 -0500
@@ -15,9 +15,9 @@ bin_PROGRAMS = $(xine_ui)
 EXTRA_PROGRAMS = session_ctrl kbindings_man
 
 kbindings_man_CFLAGS = $(AM_CFLAGS) -DKBINDINGS_MAN
-kbindings_man_LDADD = @XINE_LIBS@ $(THREAD_LIBS) \
+kbindings_man_LDADD = \
 	     $(top_builddir)/src/common/libcommon.a \
-             $(INTLLIBS)
+         $(XINE_LIBS) $(THREAD_LIBS) $(INTLLIBS)
 kbindings_man_SOURCES = kbindings_man.c kbindings_common.c
 
 xine_SOURCES = \
@@ -65,7 +65,7 @@ if HAVE_LIRC
 xine_SOURCES += lirc.c lirc.h
 endif
 
-xine_LDADD = $(XINE_LIBS) \
+xine_LDADD = \
 	     $(top_builddir)/src/xitk/xine-toolkit/libxitk.a \
              $(top_builddir)/src/xitk/Imlib-light/libImlib-light.a \
 	     $(top_builddir)/src/xitk/oxine/liboxine.a \
@@ -74,16 +74,17 @@ xine_LDADD = $(XINE_LIBS) \
 	     $(X_PRE_LIBS) $(LIRC_LIBS) $(XFT_LIBS) \
 	     $(X_LIBS) $(XTEST_LIBS) $(CURL_LIBS) $(NVTVSIMPLE_LIBS) \
 	     -lX11 -lXext -lm $(DBG_LIBS) $(XV_LIB) $(LIBPNG_LIBS) $(INTLLIBS) \
-	     $(THREAD_LIBS)
+	     $(XINE_LIBS) $(THREAD_LIBS)
 
 xine_remote_CFLAGS = $(AM_CFLAGS) -DNETWORK_CLIENT
 xine_remote_SOURCES = network.c
 xine_remote_DEPENDENCIES = $(top_builddir)/src/common/libcommon.a
-xine_remote_LDADD = $(LTLIBINTL) $(NET_LIBS) $(THREAD_LIBS) $(READLINE_LIBS) \
-	$(top_builddir)/src/common/libcommon.a $(DBG_LIBS)
+xine_remote_LDADD = \
+	$(top_builddir)/src/common/libcommon.a $(DBG_LIBS) \
+	$(LTLIBINTL) $(NET_LIBS) $(THREAD_LIBS) $(READLINE_LIBS)
 
 session_ctrl_CFLAGS = $(AM_CFLAGS)
-session_ctrl_LDADD = @XINE_LIBS@ $(top_builddir)/src/common/libcommon.a $(DBG_LIBS)
+session_ctrl_LDADD = $(top_builddir)/src/common/libcommon.a $(XINE_LIBS) $(DBG_LIBS)
 session_ctrl_SOURCES = session_common.c session_internal.h session.h ctrl_test.c
 
 noinst_HEADERS = \
