--- origsrc/xine-ui-0.99.5/src/xitk/Makefile.am	2006-07-06 12:23:16.000000000 -0500
+++ src/xine-ui-0.99.5/src/xitk/Makefile.am	2007-05-04 12:18:14.015625000 -0500
@@ -15,13 +15,13 @@
 EXTRA_PROGRAMS = session_ctrl kbindings_man
 
 kbindings_man_CFLAGS = $(AM_CFLAGS) -DKBINDINGS_MAN
-kbindings_man_LDADD = @XINE_LIBS@ 
+kbindings_man_LDADD = \
 	     $(top_builddir)/src/xitk/xine-toolkit/libxitk.a \
 	     $(top_builddir)/src/xitk/oxine/liboxine.a \
 	     $(top_builddir)/src/common/libcommon.a \
 	     $(X_PRE_LIBS) $(LIRC_LIBS) $(DYNAMIC_LD_LIBS) \
 	     $(X_LIBS) $(XTEST_LIBS) $(CURL_LIBS) $(NVTVSIMPLE_LIBS) \
-	     -lX11 -lm
+	     -lX11 -lm $(XINE_LIBS)
 kbindings_man_SOURCES = kbindings.c
 
 xine_SOURCES = \
@@ -59,7 +59,7 @@
 	videowin.c \
 	viewlog.c
 
-xine_LDADD = $(XINE_LIBS) \
+xine_LDADD = \
 	     $(top_builddir)/src/xitk/xine-toolkit/libxitk.a \
              $(top_builddir)/src/xitk/Imlib-light/libImlib-light.a \
 	     $(top_builddir)/src/xitk/oxine/liboxine.a \
@@ -67,16 +67,16 @@
              $(top_builddir)/src/common/libcommonx.a \
 	     $(X_PRE_LIBS) $(LIRC_LIBS) $(DYNAMIC_LD_LIBS) $(XFT_LIBS) \
 	     $(X_LIBS) $(XTEST_LIBS) $(CURL_LIBS) $(NVTVSIMPLE_LIBS) \
-	     -lX11 -lm $(DBG_LIBS) $(XV_LIB) $(LIBPNG_LIBS) $(INTLLIBS)
+	     -lX11 -lm $(DBG_LIBS) $(XV_LIB) $(LIBPNG_LIBS) $(XINE_LIBS) $(LTLIBINTL)
 
 xine_remote_CFLAGS = $(AM_CFLAGS) -DNETWORK_CLIENT
 xine_remote_SOURCES = network.c
 xine_remote_DEPENDENCIES = $(top_builddir)/src/common/libcommon.a
-xine_remote_LDADD = $(LTLIBINTL) $(NET_LIBS) $(THREAD_LIBS) $(READLINE_LIBS) $(NCURSES_LIB) \
-	$(top_builddir)/src/common/libcommon.a $(DBG_LIBS)
+xine_remote_LDADD = $(top_builddir)/src/common/libcommon.a $(DBG_LIBS) \
+	$(LTLIBINTL) $(NET_LIBS) $(THREAD_LIBS) $(READLINE_LIBS) $(NCURSES_LIB)
 
 session_ctrl_CFLAGS = $(AM_CFLAGS) -DCTRL_TEST
-session_ctrl_LDADD = @XINE_LIBS@ $(top_builddir)/src/common/libcommon.a $(DBG_LIBS)
+session_ctrl_LDADD = $(top_builddir)/src/common/libcommon.a $(DBG_LIBS) $(XINE_LIBS)
 session_ctrl_SOURCES = session.c
 
 noinst_HEADERS = \
